### PR TITLE
fix shop managment ui

### PR DESCRIPTION
**Pull request by nguyenminhduong**

### CODE REVIEW

File: app/src/main/java/com/example/duong/android_forder_01/ui/shopmanagementdetail/shopinformation/YourShopInformationActionHandler.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+public class YourShopInformationActionHandler {
+    private YourShopInformationContract.Presenter mListener;
+
+    public YourShopInformationActionHandler(YourShopInformationContract.Presenter listener) {
+        mListener = listener;
+    }
+
+    public void openOrderList(Shop shop) {
+        mListener.openOrderList(shop);
```
Comment (DoanVanToan) [inline app/src/main/java/com/example/duong/android_forder_01/ui/shopmanagementdetail/shopinformation/YourShopInformationActionHandler.java]: Please check null listenner, if mListener ==null appear crash

File: app/src/main/java/com/example/duong/android_forder_01/ui/shopmanagementdetail/shopinformation/YourShopInformationFragment.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.duong.android_forder_01.R;
+import com.example.duong.android_forder_01.data.model.Shop;
+import com.example.duong.android_forder_01.databinding.FragmentYourShopInformationBinding;
+
+import static com.example.duong.android_forder_01.utils.Const.KeyIntent.EXTRA_SHOP;
+
+public class YourShopInformationFragment extends Fragment
```
Comment (DoanVanToan) [inline app/src/main/java/com/example/duong/android_forder_01/ui/shopmanagementdetail/shopinformation/YourShopInformationFragment.java]: YourShopInformationFragment  should rename ShopInformationFragment

File: app/src/main/res/layout/fragment_shop_information.xml [hunk 3]
```diff
@@ -61,101 +64,174 @@
...
+
+                        <TextView
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:padding="@dimen/dp_5"
+                            android:textColor="@android:color/white"/>
+                    </TableRow>
+
+                    <TableRow android:gravity="center">
+
```
Comment (DoanVanToan) [inline app/src/main/res/layout/fragment_shop_information.xml]: remove bl, please check this file format

Comment (nguyenminhduong) [inline app/src/main/res/layout/fragment_shop_information.xml]: fixed

**Pull request merged**